src/get-started/production/index.md

```diff
@@ -14,7 +14,7 @@ This guide explains how to set up your project so you can start using the styles
 
```

Comment:
@owenatgov Can you give this page an closer look as well? There's a fair bit of dev language here that I want to make sure we're getting right.

Reply:
I'm happy with what you've done here 💪🏻